app/components/modals/modal-base.js

```diff
@@ -7,12 +7,14 @@ import { isTesting } from 'open-event-frontend/utils/testing';
 
 export default UiModal.extend({
   tagName           : 'div',
+  classNames        : ['centered-modal'],
   classNameBindings : ['isFullScreen:fullscreen', 'isSmall:small', 'isLarge:large'],
 
   openObserver: observer('isOpen', function() {
     const $element = $(this.element);
     if (this.isOpen) {
-      $element.modal('show');
+      console.log(this.defaultOptions);
+      $element.modal({...this.defaultOptions}).modal('show');
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6392563)